src/Queue.py

```diff
@@ -26,8 +27,19 @@ def getQueueLength() -> int:
     return currQueue.count(where(BallChaserKey.ID).exists())
 
 
-def isPlayerInQueue(player: Member) -> bool:
-    return currQueue.contains(where(BallChaserKey.ID) == player.id)
+def isPlayerInQueue(player: Union[Member, str]) -> bool:
+    if (isinstance(player, Member)):
+        return currQueue.contains(where(BallChaserKey.ID) == player.id)
+    elif (isinstance(player, str)):
+        member = DataFiles.currQueue.get(doc_id=int(player))
```

Comment:
I don't think you need DataFiles here. You should just be able to use currQueue like above. Also, you shouldn't need to get the player and then check to see if the player exists ? This logic seems weird. We can talk more about this if what I'm saying isn't making sense.

Reply:
Resolved.